BLSQ/iaso
# IA-3269: Project ".georegistry" should have the feature flag "Mobile: Change requests" activated

Explain what problem this PR is resolving

Related JIRA tickets : [IA-3269](https://bluesquare.atlassian.net/browse/IA-3269)

## Self proofreading checklist



## Changes

Activated feature flag **Mobile: Change requests.** for **Georegistry** project.

## How to test

From setuper folder, launch `python3 setuper.py --additional_projects` to create data and open [project page](http://localhost:8081/dashboard/settings/projects/accountId/3/) . Then check for the project **Georegistry/Géoregistre** if the feature flag **"Mobile: Change requests"** is activated by default.


## Print screen / video

[Activate-Change-Request-Feature-flag.webm](https://github.com/user-attachments/assets/1c05136a-353b-47f9-9a3c-04d3a71043cd)



[IA-3269]: https://bluesquare.atlassian.net/browse/IA-3269?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
